libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -202,126 +202,20 @@
-      "REJECT_REQUEST_WHOLESALE_SETTLEMENT": "RejectRequestWholesaleSettlement (RSM-017)",
-      "REQUEST_WHOLESALE_SETTLEMENT": "RequestWholesaleSettlement (RSM-017)",
-      "B2_C_REQUEST_WHOLESALE_SETTLEMENT": "B2CRequestWholesaleSettlement (RSM-017)",
-      "REJECT_REQUEST_FOR_REMINDERS": "RejectRequestForReminders (RSM-018)",
-      "REMINDER_OF_MISSING_MEASURE_DATA": "ReminderOfMissingMeasureData (RSM-018)",
-      "REQUEST_FOR_REMINDERS": "RequestForReminders (RSM-018)",
-      "NOTIFY_WHOLESALE_SERVICES": "NotifyWholesaleServices (RSM-019)",
-      "CONFIRM_REQUEST_SERVICE": "ConfirmRequestService (RSM-020)",
-      "REJECT_REQUEST_SERVICE": "RejectRequestService (RSM-020)",
-      "REQUEST_SERVICE": "RequestService (RSM-020)",
-      "CONFIRM_REQUEST_CHANGE_ACCOUNTING_POINT_CHARACTERISTICS": "ConfirmRequestChangeAccountingPointCharacteristics (RSM-021)",
-      "REJECT_REQUEST_CHANGE_ACCOUNTING_POINT_CHARACTERISTICS": "RejectRequestChangeAccountingPointCharacteristics (RSM-021)",
-      "REQUEST_CHANGE_ACCOUNTING_POINT_CHARACTERISTICS": "RequestChangeAccountingPointCharacteristics (RSM-021)",
-      "ACCOUNTING_POINT_CHARACTERISTICS": "AccountingPointCharacteristics (RSM-022)",
-      "CONFIRM_REQUEST_CANCELLATION": "ConfirmRequestCancellation (RSM-024)",
-      "REJECT_REQUEST_CANCELLATION": "RejectRequestCancellation (RSM-024)",
-      "REQUEST_CANCELLATION": "RequestCancellation (RSM-024)",
-      "NOTIFY_CANCELLATION": "NotifyCancellation (RSM-025)",
-      "CONFIRM_REQUEST_CHANGE_CUSTOMER_CHARACTERISTICS": "ConfirmRequestChangeCustomerCharacteristics (RSM-027)",
-      "REJECT_REQUEST_CHANGE_CUSTOMER_CHARACTERISTICS": "RejectRequestChangeCustomerCharacteristics (RSM-027)",
-      "REQUEST_CHANGE_CUSTOMER_CHARACTERISTICS": "RequestChangeCustomerCharacteristics (RSM-027)",
-      "CHARACTERISTICS_OF_A_CUSTOMER_AT_AN_AP": "CharacteristicsOfACustomerAtAnAP (RSM-028)",
-      "CONFIRM_REQUEST_CHANGE_BILLING_MASTER_DATA": "ConfirmRequestChangeBillingMasterData (RSM-030)",
-      "REJECT_REQUEST_CHANGE_BILLING_MASTER_DATA": "RejectRequestChangeBillingMasterData (RSM-030)",
-      "REQUEST_CHANGE_BILLING_MASTER_DATA": "RequestChangeBillingMasterData (RSM-030)",
-      "NOTIFY_BILLING_MASTER_DATA": "NotifyBillingMasterData (RSM-031)",
-      "REJECT_REQUEST_BILLING_MASTER_DATA": "RejectRequestBillingMasterData (RSM-032)",
-      "REQUEST_BILLING_MASTER_DATA": "RequestBillingMasterData (RSM-032)",
-      "CONFIRM_REQUEST_CHANGE_OF_PRICELIST": "ConfirmRequestChangeOfPricelist (RSM-033)",
-      "REJECT_REQUEST_CHANGE_OF_PRICELIST": "RejectRequestChangeOfPricelist (RSM-033)",
-      "REQUEST_CHANGE_OF_PRICELIST": "RequestChangeOfPricelist (RSM-033)",
-      "NOTIFY_PRICELIST": "NotifyPricelist (RSM-034)",
-      "REJECT_REQUEST_PRICELIST": "RejectRequestPricelist (RSM-035)",
-      "REQUEST_PRICELIST": "RequestPricelist (RSM-035)"
+      "NOTIFY_AGGREGATED_MEASURE_DATA": "Notifikation om aggregerede tidsserier (RSM-014)",
```

Comment:
nit: can we have to same sortin?
that is, our message has the prefixes (NOTIFY,...) in the same order :)

Reply:
If you agree to this, give "libs/dh/globalization/assets-localization/src/assets/i18n/en.json" a look :)

Reply:
Good idea to align sorting 🙂 Question is, do we actually want alphabetical sorting in the UI dropdown? It used to be sorted by "RSM" number, which is also how it is sorted in the translation files (this has no effect on the UI though).

Reply:
Meyer says to go with RSM number ordering for now. So I might take a different approach here, keep the order as it is in the translation files and then let the translation files dictate the order. Alternatively we could change the order in the backend client, or even in the BFF, but I'm not sure if that's any better.